domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/vilkår/kvp/FørstegangsbehandlingKvpEx.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.tiltakspenger.saksbehandling.domene.vilkår.kvp
+
+import no.nav.tiltakspenger.saksbehandling.domene.behandling.Førstegangsbehandling
+
+fun Førstegangsbehandling.leggTilKvpSaksopplysning(
```

Comment:
Hvorfor lage en extension function for en klasse vi selv eier og har fullstendig kontroll over?

Reply:
Jeg fant svaret selv..

Reply:
Ja, jeg synes det er en fin måte å gruppere domenet på. Her kan da kvp-ting bo i kvp-mappa.